systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -943,6 +945,59 @@ void testMirrorMakerTlsScramSha() {
+        String coPodName = KUBE_CLIENT.listResourcesByLabel("pod", "name=strimzi-cluster-operator").get(0);
+        resources().kafkaEphemeral(CLUSTER_NAME, 1).done();
+
+        // rolling update for kafka
+        operationID = startTimeMeasuring(Operation.ROLLING_UPDATE);
+        // set annotation to trigger Kafka rolling update
+        CLIENT.apps().statefulSets().inNamespace(NAMESPACE).withName(kafkaClusterName(CLUSTER_NAME)).cascading(false).edit()
+                .editMetadata()
+                    .addToAnnotations("strimzi.io/manual-rolling-update", "true")
+                .endMetadata().done();
+
+        // check annotation to trigger rolling update
+        assertTrue(Boolean.parseBoolean(CLIENT.apps().statefulSets().inNamespace(NAMESPACE).withName(kafkaClusterName(CLUSTER_NAME))
+                .get().getMetadata().getAnnotations().get("strimzi.io/manual-rolling-update")));
+
+        // wait when annotation will be removed
+        waitFor("CO removes rolling update annotation", WAIT_FOR_ROLLING_UPDATE_INTERVAL, WAIT_FOR_ROLLING_UPDATE_TIMEOUT,
+            () -> getAnnotationsForSS(NAMESPACE, kafkaClusterName(CLUSTER_NAME)) == null
+                || !getAnnotationsForSS(NAMESPACE, kafkaClusterName(CLUSTER_NAME)).containsKey("strimzi.io/manual-rolling-update"));
+
+        // check rolling update messages in CO log
+        String coLog = KUBE_CLIENT.logs(coPodName);
+        assertThat(coLog, containsString("Rolling Kafka pod " + kafkaClusterName(CLUSTER_NAME) + "-0" + " due to manual rolling update"));
+
+        // rolling update for zookeeper
+        operationID = startTimeMeasuring(Operation.ROLLING_UPDATE);
+        // set annotation to trigger Zookeeper rolling update
+        CLIENT.apps().statefulSets().inNamespace(NAMESPACE).withName(zookeeperClusterName(CLUSTER_NAME)).cascading(false).edit()
+                .editMetadata()
+                    .addToAnnotations("strimzi.io/manual-rolling-update", "true")
+                .endMetadata().done();
+
+        // check annotation to trigger rolling update
+        assertTrue(Boolean.parseBoolean(CLIENT.apps().statefulSets().inNamespace(NAMESPACE).withName(zookeeperClusterName(CLUSTER_NAME))
+                .get().getMetadata().getAnnotations().get("strimzi.io/manual-rolling-update")));
+
+        // wait when annotation will be removed
+        waitFor("CO removes rolling update annotation", WAIT_FOR_ROLLING_UPDATE_INTERVAL, WAIT_FOR_ROLLING_UPDATE_TIMEOUT,
+            () -> getAnnotationsForSS(NAMESPACE, zookeeperClusterName(CLUSTER_NAME)) == null
+                || !getAnnotationsForSS(NAMESPACE, zookeeperClusterName(CLUSTER_NAME)).containsKey("strimzi.io/manual-rolling-update"));
+
+        // check rolling update messages in CO log
+        coLog = KUBE_CLIENT.logs(coPodName);
+        assertThat(coLog, containsString("Rolling Zookeeper pod " + zookeeperClusterName(CLUSTER_NAME) + "-0" + " to manual rolling update"));
```

Comment:
One nit. Does using `assertThat` have any reason? Why you do not use 
assertTrue(coLog.contains("Rolling Zookeeper pod " + zookeeperClusterName(CLUSTER_NAME) + "-0" + " to manual rolling update"));` ?

Reply:
The value of `assertThat` + `containsString` is that when ST fails you can see the actual log and expected log. in case with `assertTrue` you will see just expected `true` but `false`